app/controllers/v0/institutions_controller.rb

```diff
@@ -42,18 +41,16 @@ def show
     def downcase_search_query_params!
       %i(type_name student_veteran_group
          yellow_ribbon_scholarship principles_of_excellence
-         eight_keys_to_veteran_success caution).each do |k|
+         eight_keys_to_veteran_success caution country).each do |k|
         params[k].try(:downcase!)
       end
-      %i(country state).each do |k|
-        params[k].try(:upcase!)
-      end
+      params[:state].try(:upcase!)
       params[:name].try(:strip!)
```

Comment:
So I don't think it's good practice to modify params. It would be better to use an instance variable. As a user, if I pass in state as lower case, if it fails validation, I should see the params exactly as I provided them.